json_validation.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+
+"""
+This script takes a JSON file and runs validation against a JSON schema. If no schema file is given,
+ it checks against the ArrayExpress submission schema.
+"""
+
+import argparse
+
+from validator.json_schema_validation import validate_submission_json
+from utils.common_utils import file_exists
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('json',
```

Comment:
I think convention when using the add_argument bit is to provide options in the form that you're using for the following argument (the schema), so providing a short cut for the argument (-j) and a longer doubled dashed one version of it (--json). I would suggest though to be more explicit at this point on what is the purpose of the json (like json for USI input data), as in the future the script might receive other jsons and then telling them apart will be more painful (where as now you don't have any API commitments to comply as this is the first version).

Reply:
Yes, being explicit is always good. But I'll leave this for now as it is meant as a simple general-purpose script (with one mandatory input file).